hail/scripts/deploy.sh

```diff
@@ -18,11 +18,15 @@ WHEEL=$5
 GITHUB_OAUTH_HEADER_FILE=$6
 HAIL_GENETICS_HAIL_IMAGE=$7
 HAIL_GENETICS_HAILTOP_IMAGE=$8
-WHEEL_FOR_AZURE=$9
-WEBSITE_TAR=${10}
+QOB_VEP_GRCH37_IMAGE=$9
+QOB_VEP_GRCH38_IMAGE=${10}
```

Comment:
Let's use HAIL_GENETICS_VEP_85_GRCH37_IMAGE and likewise for consistency with the other images.